src/cdk/drag-drop/drag-ref.ts

```diff
@@ -468,7 +470,10 @@ export class DragRef<T = any> {
    * @param handle Handle element to be enabled.
    */
   enableHandle(handle: HTMLElement) {
-    this._disabledHandles.delete(handle);
+    if (this._disabledHandles.has(handle)) {
+      this._disabledHandles.delete(handle);
+      toggleNativeDragInteractions(handle, this.disabled);
```

Comment:
should this say `false` instead of `this.disabled`?

Reply:
No, the handle should stay disabled if it is enabled while the drag item is disabled.

Reply:
Then should it also use `this.disabled` in `disableHandle`? seems odd that the methods aren't mirrored

Reply:
No since `disableHandle` only disables the handle, not the entire draggable item. A handle is considered "inactive" if either it is disabled or the item is disabled. This might be slightly confusing since `toggleNativeDragInteractions(true)` means that interactions should be enabled which is true when dragging is disabled.

Reply:
Ok I think I understand it, just a little confusing at first sight